benchmarks/benchmark7.ipynb

```diff
@@ -10,7 +10,7 @@
       "application/javascript": [
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    print("Compare generated expression for S:")
</pre><p>Why not just use the markdown cells for the text? Then the font won't be weird.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/usnistgov/pfhub/pull/1504/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>

Reply:
Sorted in 88099cc